### PR TITLE
[ISSUE #282]Translation for "Deployment-RocketMQ Dashboard" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #282: Translation for "Deployment-RocketMQ Dashboard" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "Deployment-RocketMQ Dashboard" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #282 

## Brief changelog

- Translation for "Deployment-RocketMQ Dashboard" Section in the v4.x & 5.0 Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md [hunk 1]
```diff
@@ -1,147 +1,150 @@
...
+| --------- | ------------------------------------------------------------ |
+| OPS       | Modify nameserver address; use `VIPChannel`                  |
+| Dashboard | Check broker, topic message volume                           |
+| Cluster   | Cluster distribution, broker configuration, runtime information |
+| Topic     | Search, filter, delete, update/add topics, message routing, send messages, reset consumption points |
+| Consumer  | Search, delete, add/update consumer groups, terminals, consumption details, configuration |
+| Message   | Message records, private messages, message trace, etc. message details |
 
-操作面板：
+operation panel：
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md]: operation -> Operation

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md [hunk 2]
```diff
@@ -1,147 +1,150 @@
...
 
 ![1657630174311](../picture/17Dashboard/1_dashboard.png)
 
-## 快速开始
+## Quick start
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md]: Quick start -> Quick Start

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md [hunk 3]
```diff
@@ -1,147 +1,150 @@
...
 
-① 安装docker，拉取 ```rocketmq-dashboard``` 镜像
+① Install docker and pull the rocketmq-dashboard image
 
 ```shell
 docker pull apacherocketmq/rocketmq-dashboard:latest
 ```
 
-② docker 容器中运行 ```rocketmq-dashboard```
+② Run in a docker container ```rocketmq-dashboard```
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md]: docker -> Docker

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md [hunk 4]
```diff
@@ -1,147 +1,150 @@
...
+Browser page access：namesrv.addr:8080
 
-关闭 ```rocketmq-dashboard``` : ctrl + c
+Close ```rocketmq-dashboard``` : ctrl + c
 
-再次启动：执行 ②
+Restart：execution ②
 
-**tips**：下载后的源码需要上传到 Linux 系统上编译，本地编译可能会报错。
+**tips**：The downloaded source code needs to be uploaded to the Linux system for compilation, and local compilation may give errors
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md]: give -> return

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md [hunk 5]
```diff
@@ -1,147 +1,150 @@
...
+
+- Cluster consumption supports resetting consumption offsets, but broadcast mode does not.
+- If a consumer is not online, it is not possible to reset the consumption offset.
 
-- 集群消费支持重置消费位点, 广播模式不支持.
-- 消费者不在线不能重置消费位点
+:::
 
-### 4. 扩容 Topic 队列
+### 4. Expand topic queue
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/17Dashboard.md]: queue -> queues

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Great work @mxsm , just a few change requests for you. Remind that also correct them in the 5.0 version doc.

**Comment by mxsm** [comment]
@tsunghanjacktsai  I will fix it and re-submit later

**Comment by mxsm** [comment]
@tsunghanjacktsai  PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**